kostal/README.md

```diff
@@ -1,14 +1,21 @@
 # KOSTAL
 
+VERSION = "1.3.1"
+
 # Requirements
```

Comment:
Just created a new thread for this plugin. URL is:
https://knx-user-forum.de/forum/supportforen/smarthome-py/1109697-kostal-plugin-piko-wechselrichter